### PR TITLE
Warmboot KVM tests - replace docker-exec 1s timeout with 5s

### PR DESCRIPTION
**Pull request by vaibhavhd**
### Description of PR

Summary: Warm reboot tests are failing in PR checks. Increase docker-exec timeout from 1s to 5s for KVM testing.

Fixes: https://github.com/Azure/sonic-mgmt/issues/3264

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Approach
Find the path to warm-reboot script.
Replace 1s docker exec timeout with 5s.

#### What is the motivation for this PR?
Fix PR test failures.

Failure is due to 1s timeout set here for the warmboot pre-reboot-check
https://github.com/Azure/sonic-utilities/blob/master/scripts/fast-reboot#L322

#### How did you verify/test it?
Tested on local KVM. Wait for PR results for this PR.

### CODE REVIEW

File: tests/common/fixtures/advanced_reboot.py [hunk 1]
```diff
@@ -40,6 +40,14 @@ def __init__(self, request, duthost, ptfhost, localhost, tbinfo, creds, **kwargs
         )
 
         if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
```
Comment (yxieca) [inline tests/common/fixtures/advanced_reboot.py:42]: Please add a detailed comments here to state why this change is needed.

**Review by yxieca — APPROVED** [review]

File: tests/common/fixtures/advanced_reboot.py (hunk 1 above)
Comment (vaibhavhd) [reply tests/common/fixtures/advanced_reboot.py:42]: Updated now, thanks!
